Kabalisa/Send-IT-API
# #162364252 add foreign key relationship

#### What does this PR do?
add a foreign key relationship between users and parcels tables.
#### Description of Task to be completed?
* drop the current tables
* delete the id column from users table
* make the userid column in the users table a primary key
* make the userid column in the parcels table a foreign key referencing from the users table
* restructure some queries where necessary
#### How should this be manually tested?
clone this repository with ```git clone```
run ```npm start```
write the signup endpoint's URL in POSTMAN.
create two similar users with same userids to check for the error you will get. 
#### Any background context you want to provide?
This correction of a bug is carried out on a bug branch.
#### What are the relevant pivotal tracker stories?
[#162364252](https://www.pivotaltracker.com/story/show/162364252)
